tests/yast2_gui/yast2_security.pm

```diff
@@ -53,11 +51,8 @@ sub run {
     wait_still_screen 3;
 
     # Check previously set values + Miscellaneous Settings
-    y2_module_guitest::launch_yast2_module_x11("security", match_timeout => 120);
-    if (is_sle('=15-SP4')) {
-        record_soft_failure('bsc#1204176 - Resizing window as workaround for YaST content not loading');
-        send_key_until_needlematch('yast2_security-login-settings', 'alt-f10', 10, 2);
-    }
+    y2_module_guitest::launch_yast2_module_x11("security", match_timeout => 120) if (is_sle('>=15-SP4'));
+    apply_workaround_bsc1204176('yast2_security-login-settings');
```

Comment:
yep, we missed this `if`.

Reply:
you mean the `if` should have been on the `apply_workaround` line, not the `y2_module_guitest::…`, right?

Reply:
Yes, the if should be trigger apply_workaround_bsc1204176 only for SLE '>=15-SP4'. 
Made a fix, VR is currently running. PR will be made when VR is successful.

Reply:
VR failed. The problem was that we didn't "miss" the check, it was accidentially at the end of the line above, so calling yast2_module_x11 never happened on Tumbleweed. 
Fixed that, restarting VR.

Reply:
which is what I wrote above - but maybe not clear enough. sorry for that